frontend/src/components/common/authorized-menu/navigation-menu/navigation-menu.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, SubNavigationMenuItem } from 'common/types/types';
+import { getValidClasses } from 'helpers/helpers';
+import { useMatch, useResolvedPath } from 'hooks/hooks';
+
+import { NavigationMenuItem } from './navigation-menu-item/navigation-menu-item';
+import styles from './styles.module.scss';
+
+type Props = {
+  name: string;
+  subroutes: SubNavigationMenuItem[];
+};
+
+const NavigationMenu: FC<Props> = ({ name, subroutes }) => {
+  return (
+    <div>
+      <h4 className={styles.title}>{name}</h4>
+      <div className={getValidClasses(styles.links, styles.bottomLine)}>
+        {subroutes.map(({ name: routeName, iconName, href }) => {
+          const resolvedPath = useResolvedPath(href);
+          const isCurrentRoute = useMatch({
+            path: resolvedPath.pathname,
+            end: true,
+          });
```

Comment:
Let's provide boolean value, as on NavigationMenuItem it is used only as boolean
```suggestion
          const isCurrentRoute = Boolean(useMatch({
            path: resolvedPath.pathname,
            end: true,
          }));
```